zeroeq/http/server.h

```diff
@@ -93,9 +93,17 @@ class Server : public zeroeq::Receiver
 
     /** @name Object registration for PUT and GET requests */
     //@{
-    /** Handle PUT and GET for the given object. */
-    bool handle( servus::Serializable& object )
-        { return handlePUT( object ) && handleGET( object );}
+    /**
+     * Handle PUT and GET for the given object.
+     *
+     * @param object the object to update and serve on receive()
+     * @param event uses this as the URL endpoint instead of the default
```

Comment:
should be 'use', not 'uses'. I also don't find 'event' very clear, but I don't know have a good suggestion. maybe 'address' or something like 'httpPath'?